demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -317,6 +316,113 @@ export function ProductsGrid({ toolbarAction, rowAction, actionsColumnWidth = 52
             flex: 1,
             minWidth: 150,
         },
+        {
+            field: "nestedGroups",
+            headerName: intl.formatMessage({ id: "product.nestedGroups", defaultMessage: "Nested groups" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                const secondaryTextNestedItem3GroupValues: string[] = [
+                    typeof row.price === "undefined" || row.price === null
+                        ? ""
+                        : intl.formatNumber(row.price, { minimumFractionDigits: 2, maximumFractionDigits: 2, style: "currency", currency: "EUR" }),
+                    row.category?.title ?? "",
+                ];
+                const secondaryTextGroupValues: string[] = [
+                    typeof row.price === "undefined" || row.price === null
+                        ? ""
+                        : intl.formatNumber(row.price, { minimumFractionDigits: 2, maximumFractionDigits: 2, style: "currency", currency: "EUR" }),
+                    row.category?.title ?? "",
+                    secondaryTextNestedItem3GroupValues.filter(Boolean).join(" • "),
+                ];
```

Comment:
This is complicated to read (and probably) maintain. Should we add a component to `@comet/admin` to help with this? I'm thinking of something like this:

```tsx
<CombinedText 
  separator="°"
  segments={[row.price,  row.category?.title]}
/>
```

(ignore the naming for now)

Reply:
I'm not sure how easy this would be, as theoretically groups can be nested infinitely inside other types (`group`, `formattedMessage`). Rendering a react component might cause some issues. 

I could try this out in a follow-up task though, if you like.